core/math/random_number_generator.h

```diff
@@ -59,7 +59,10 @@ class RandomNumberGenerator : public Reference {
 
 	_FORCE_INLINE_ int randi_range(int from, int to) {
 		unsigned int ret = randbase.rand();
-		return ret % (to - from + 1) + from;
+		if (to < from)
```

Comment:
Or simpler: `return ret % (Math::abs(to - from) + 1) + to;`

Reply:
You mean changing the whole condition into one line? Then you probably need something more like
`return ret % (Math::abs(to - from) + 1) + Math::min(from, to);`
which does one more check.

Reply:
Indeed, not worth it then.